.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules
 .auth.json
 _cache.db
 all*.json
+config.json
```

Comment:
I run `bin/sync` config.json` like it says in the README for example repos with settings that are different than `projects/loopback.json` is using. I should probably merge them together at some point.